lighthouse-core/report/templates/report-template.html

```diff
@@ -61,6 +61,18 @@
             <h1 class="menu__header-title">Lighthouse</h1>
             <div class="menu__header-version">Version: {{lighthouseVersion}}</div>
           </div>
+          {{#each relatedReports}}
```

Comment:
Like the config panel, I don't see us using this outside of an interactive mode like perfX. This one will be slightly more awkward, but possible to pull out into a partial?

There might be some fancy handlebars way of including it, but also could do it brute force, something like
```html
{{#if_eq reportContext "perf-x" }}
  {{> related-reports}}
{{else}}
  <ul class="menu__nav">
    {{#each aggregations}}
    <li class="menu__nav-item">
      <a class="menu__link" href="#{{nameToLink this.name}}">
        {{ this.name }}
      </a>
    </li>
    {{/each}}
  </ul>
{{/if_eq}}
```
(including the `menu__nav` block in the `related-reports` partial)